docs/tutorials/tut-integration-ui.md

```diff
@@ -73,7 +73,7 @@ The following is an example of the proxy settings filled out:
 
 ![screen shot 2019-01-08 at 13 38 33](../doc_imgs/tutorials/tut-integration-ui/parameter-proxy.png)
 
->Next we will add the insecure setting called “insecure”. This will also be a boolean. Set the initial value to “false” as well and we will write “Trust any cert”.
+>Next we will add the insecure setting called “insecure”. This will also be a boolean. Set the initial value to “false” as well and we will write "Allow any cert”.
```

Comment:
```suggestion
>Next we will add the insecure setting called “insecure”. This will also be a boolean. Set the initial value to “false” as well and we will write ״Allow any cert״.
```